packages/core/useDraggable/index.ts

```diff
@@ -135,9 +142,10 @@ export function useDraggable(target: MaybeComputedRef<HTMLElement | SVGElement |
   }
 
   if (isClient) {
-    useEventListener(draggingHandle, 'pointerdown', start, true)
-    useEventListener(draggingElement, 'pointermove', move, true)
-    useEventListener(draggingElement, 'pointerup', end, true)
+    const useCapture = options.useCapture || options.useCapture === null
+    useEventListener(draggingHandle, 'pointerdown', start, useCapture)
+    useEventListener(draggingElement, 'pointermove', move, useCapture)
+    useEventListener(draggingElement, 'pointerup', end, useCapture)
```

Comment:
Normalizing `useCapture` fixes the typescript error which occurs because `useEventListener` does not accept a `MaybeComputedRef` as an `options` parameter
```suggestion
    useEventListener(draggingHandle, 'pointerdown', start, resolveUnref(useCapture))
    useEventListener(draggingElement, 'pointermove', move, resolveUnref(useCapture))
    useEventListener(draggingElement, 'pointerup', end, resolveUnref(useCapture))
```


Though note, that this is not reactive. You could either make it reactive here, or make `options` reactive in `useEventListener`, or use a plain optional boolean instead of a `MaybeComputedRef` for the `useCapture` parameter.

Reply:
Thanks! I'll go with your suggestion, that seems consistent with how the other optional booleans are handled here.

Reply:
The other optional booleans in this composable are however reactive, as they are normalized within callback functions. I worry that using `MaybeComputedRef` for an option without it being reactive may set false expectations when using the composable.

Reply:
I don't think you can change the capture mode of an event listener, so should I go with a plain optional boolean ?

Reply:
Since `useEventListener`s capture mode isn't reactive, I think this composables shouldn't be as well for now. So that approach should be fine.